service/framework.go

```diff
@@ -653,3 +661,188 @@ func newPodFramework(config Config) (*framework.Framework, error) {
 
 	return podFramework, nil
 }
+
+func migrateTPRsToCRDs(logger micrologger.Logger, clientSet *versioned.Clientset) {
+	logger.Log("debug", "start TPR migration")
+
+	var err error
+
+	// List all TPOs.
+	var b []byte
+	{
+		e := "/apis/cluster.giantswarm.io/v1/namespaces/default/kvms"
+		b, err = clientSet.Discovery().RESTClient().Get().AbsPath(e).DoRaw()
+		if err != nil {
+			logger.Log("error", fmt.Sprintf("%#v", err))
+			return
+		}
+
+		fmt.Printf("\n")
+		fmt.Printf("b start\n")
+		fmt.Printf("%s\n", b)
+		fmt.Printf("b end\n")
+		fmt.Printf("\n")
```

Comment:
I think you forgot to remove that.

Reply:
I would like to keep this for the migration. I want to prepare everything and just have some PRs where I enable some commented code and then I am done.

Reply:
This is removed after the migration anyway.

Reply:
And the amount of data printed does not harm in our case.